src/catkin_pkg/package.py

```diff
@@ -371,7 +373,7 @@ def parse_package(path, warnings=None):
     # Force utf8 encoding for python3.
     # This way unicode files can still be processed on non-unicode locales.
     kwargs = {}
-    if sys.version_info.major >= 3:
+    if _py3:
```

Comment:
Since this is only being used once please move the condition to here instead of using a variable.

Since we only care about the major version can you please update it to only check for `[0] >= 3`.